judo-runtime-core-dao-rdbms/src/main/java/hu/blackbelt/judo/runtime/core/dao/rdbms/AbstractRdbmsDAO.java

```diff
@@ -197,6 +197,17 @@ protected Payload create(EClass eClass, Payload payload, QueryCustomizer<ID> que
         }
     }
 
+    @Override
+    public List<Payload> createAll(EClass eClass, Iterable<Payload> payloads, QueryCustomizer<ID> queryCustomizer) {
+        List resultPayloads = new ArrayList<>();
+
+        for (Payload payload : payloads) {
+            resultPayloads.add(create(eClass, payload, queryCustomizer));
+        }
+
+        return ImmutableList.copyOf(resultPayloads);
```

Comment:
Why are we wrapping the result? The return type is a basic `List`. Returning an immutable will yield potential runtime errors.

Reply:
In the SDK, we often create an immutable list from a mutable one. I've done it here, but I can remove it. I'll ask @robertcsakany .